packages/runtime/src/runtimes/react/utils/suppress-ref-warning.ts

```diff
@@ -0,0 +1,23 @@
+const originalError = console.error
+let patched = false
+
+/**
+ * @see https://github.com/facebook/react/blob/a2505792ed17fd4d7ddc69561053c3ac90899491/packages/react-reconciler/src/ReactFiberBeginWork.new.js#L1814-L1890
+ */
+export function suppressRefWarning(ownerName: string): void {
```

Comment:
I thought about co-locating this with its usage as it's only being used once but wanted to get consensus around it.

Reply:
It's funny but this is actually not doing anything right now because where we pass the ref changed when we introduced `<RenderHook>`. The plan is to fix this once we refactor that out. As for whether this is colocated or not, don't have a strong preference there.

Reply:
I'll leave it then!